MANIFEST.in

```diff
@@ -0,0 +1,8 @@
+include README.md LICENSE settings.yml
+recursive-include buildtest *
```

Comment:
Why have this line with line 3? Seems like duplicate work but also adding all ``*.yml`` files

Reply:
I would usually do lines that add specific types, but since there is this new command graft, I think that's sufficient. I do need to still look it up to learn about the details.